animation_nodes/data_structures/gpencils/gpencil_stroke_data.pyx

```diff
@@ -0,0 +1,42 @@
+# cython: profile=True
+import textwrap
+from .. lists.base_lists cimport DoubleList, Vector3DList
+
+class GPStroke:
+    def __init__(self, vertices = None, strength = None, pressure = None,
+        uv_rotation = None, line_width = None, draw_cyclic = None, start_cap_mode = None,
+        end_cap_mode = None, material_index = None, display_mode = None):
+
+        if vertices is None: vertices = Vector3DList()
+        if strength is None: strength = DoubleList()
+        if pressure is None: pressure = DoubleList()
+        if uv_rotation is None: uv_rotation = DoubleList()
+        if line_width is None: line_width = 250
+        if draw_cyclic is None: draw_cyclic = False
+        if start_cap_mode is None: start_cap_mode = "ROUND"
+        if end_cap_mode is None: end_cap_mode = "ROUND"
+        if material_index is None: material_index = 0
+        if display_mode is None: display_mode = "SCREEN"
+
+        self.vertices = vertices
+        self.strength = strength
+        self.pressure = pressure
+        self.uv_rotation = uv_rotation
```

Comment:
Use plural nouns. `stength` -> `strengths`. The same for all list attributes.

Reply:
Okay.